content/overview/security/conforming-federal-security-regulations.md

```diff
@@ -11,22 +11,22 @@ aliases:
 
-With the cloud.gov PaaS, we have fundamental technological assurances that each new system cannot alter or change our desired AWS configuration. This allows us to enforce a single control layer across all systems with significantly less effort. Instead, we spend more time focusing on the web facing  portion of a system, which is most often the source of attacks anyway.
+The cloud.gov PaaS provides fundamental technological assurances that each new system cannot alter or change our desired AWS configuration. Enforce this single control layer across all systems takes significantly less effort overall. Instead, teams spend more time focusing on the web facing portion of a system, which is most often the source of attacks anyway.
 
 ## Helping system developers reduce effort needed to gain ATO
 
-Once a system is deployed on cloud.gov, partner agencies can use our security documentation in a machine-readable format. Since GSA is the same organization running the FedRAMP program, there should be a high level of trust that we have implemented the controls we say we’ve implemented. That said, the code of all of our systems is open-source, so agencies can directly inspect what we’ve done.
+Once a system is deployed on cloud.gov, partner agencies can use the cloud.gov security documentation in a machine-readable format. GSA runs the FedRAMP program and owns cloud.gov, so there should be a high level of trust that the controls are implemented. That said, the code of all of our systems is open-source, so agencies can directly inspect what we’ve done.
 
-After an agency has accepted the controls implemented on the IaaS and PaaS layers, they can focus their attention where it matters most: on the application itself. We have a list of roughly 24 NIST recommended controls that should always be evaluated when creating a web system. This is a huge time-saver, and will directly increase the security of an system since more time will be spent on the most important controls.
+After an agency has accepted the controls implemented on the IaaS and PaaS layers, they can focus their attention on the product itself they are deploying. The cloud.gov team has a list of roughly 24 NIST-recommended controls that you should always evaluate when creating a web system. This huge time saver directly increases the security of a system, since it leads teams to spend more time on the most important controls.
 
-We’re not going to stop there though - we will have trusted non-governmental third-parties evaluate our work as well. Updates on that process will be forthcoming.
+cloud.gov doesn’t stop there, though. Trusted non-governmental third-parties will evaluate the platform, as well. Updates on that process are forthcoming.
```

Comment:
"Trusted non-governmental third-parties will evaluate the platform, as well. Updates on that process are forthcoming." We do this! Suggest "An accredited Third Party Assessment Organization (3PAO) also evaluates the platform as part of the FedRAMP P-ATO authorization process."